source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Custom Certificates on the Global CDN
+description: For contract customers who need to retain custom TLS certificates, bring your own certificate to the Global CDN to replace legacy HTTPS service.
+earlyaccess: true
+earlynote: This documentation covers features and options not available across the entire platform.
+---
+
+## Access
+
+A white glove concierge service is now available and limited to contract customers, including **Enterprise**, **EDU+**, **Pantheon One**, **Elite**, and **Reseller**, who are using Legacy SSL.
+
+## Upgrade to the Global CDN
+
+If your site is using our Legacy SSL service it's on deprecated, legacy infrastructure, served from a single server in the central US. When you upgrade to the Global CDN you'll see faster performance, with content  delivered from over 50 points of presence (POPs) around the world.
+
+If your site uses Legacy SSL, it's also on an outdated TLS configuration. The Global CDN is configured to only use TLS 1.2 and no weak 3DES cipher. On the legacy infrastructure, your site isn't as fast, secure, or as resilient as it could be.
+
+### Option 1: Automated Certificate Management via Let's Encrypt
```

Comment:
I wonder if we need this section - I'd suspect it's likely if a customer made it to this doc, they've already ruled out this option, and removing this section would keep focus on the process for using their own cert.

Reply:
Good point. Making a note to circle back on this.